lib/phoenix/controller.ex

```diff
@@ -884,6 +884,9 @@ defmodule Phoenix.Controller do
       Defaults to none
     * `:offset` - the bytes to offset when reading. Defaults to `0`
     * `:length` - the total bytes to read. Defaults to `:all`
+    * `:encode` - encode filename using URI.encode_www_form.
+      Defaults to `nil`. Any value different of `false` will apply
```

Comment:
I see what you mean. So I think the `nil` as default is confusing. The default should be `true`, we should always encode if `true`, unless set to `false`. We should also say that disabling  it may have security implications.